0000-template.md

```diff
@@ -1,46 +1,99 @@
-- Feature Identifier: (unique identifier for this proposed feature, my_new_feature)
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
-- Affects Ara System: (ARAid, AFS, Other etc.)
-- RFC PR: (leave this empty)
-- Ara Issue: (leave this empty)
+<Topic Name>
+==============
 
-# Summary
+---
+Request For Comments: 0000
+---
+Author(s): <Author(s) First & Last>
+---
+Category: <Category>
```

Comment:
@jwerle Are categories arbitrary? As in will we have a predefined list of categories?

Reply:
@cckelly yeah I imagine this to be somewhat like tags